source/documents/insert.txt

```diff
@@ -79,6 +79,10 @@ To insert documents into your collection:
                     { "_id" : 8751, "title" : "The Banquet", "author" : "Dante", "copies" : 2 }
                   ]
 
+            .. note::
+
+              If an object ID is not provided by user, Compass will automatically generate an object ID.
```

Comment:
We default to present tense when possible, so instead of

"Compass will automatically generate an objectId"

we can say

"Compass automatically generates an ObjectId"